Mage.Sets/src/mage/cards/c/CharismaticConqueror.java

```diff
@@ -0,0 +1,144 @@
+package mage.cards.c;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.abilities.keyword.VigilanceAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Zone;
+import mage.filter.FilterPermanent;
+import mage.filter.common.FilterArtifactCreaturePermanent;
+import mage.filter.predicate.permanent.TappedPredicate;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.game.permanent.token.IxalanVampireToken;
+import mage.players.Player;
+import mage.target.targetpointer.FixedTarget;
+
+/**
+ * Charismatic Conqueror {1}{W}
+ * Creature - Vampire Soldier 2/2
+ * Vigilance
+ * Whenever an artifact or creature enters the battlefield untapped and under an opponent's control, they may tap that permanent. If they don't, you create a 1/1 white Vampire creature token with lifelink.
+ *
+ * @author DominionSpy
+ */
+public final class CharismaticConqueror extends CardImpl {
+
+    private static final FilterPermanent filter = new FilterArtifactCreaturePermanent();
+
+    static {
+        filter.add(TappedPredicate.UNTAPPED);
+    }
+
+    public CharismaticConqueror(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{W}");
+
+        this.subtype.add(SubType.VAMPIRE);
+        this.subtype.add(SubType.SOLDIER);
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(2);
+
+        // Vigilance
+        this.addAbility(VigilanceAbility.getInstance());
+
+        // Whenever an artifact or creature enters the battlefield untapped and under an opponent's control, they may tap that permanent. If they don't, you create a 1/1 white Vampire creature token with lifelink.
+        this.addAbility(new CharismaticConquerorTriggeredAbility());
+//        this.addAbility(new EntersBattlefieldOpponentTriggeredAbility(
+//                Zone.BATTLEFIELD, new CharismaticConquerorEffect(), filter, false, SetTargetPointer.PERMANENT
+//        ));
```

Comment:
okay, so why did you choose to make a custom ability rather than use the common class?

Reply:
The ability was not triggering and I couldn't work out why. I see now that the filter is for an artifact creature, not an artifact OR creature. Will fix.